hack/mac-registry-tunnel.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+
+#
+# .-'_.---._'-.
+# ||####|(__)||   Protect your secrets, protect your business.
+#   \\()|##//       Secure your sensitive data with Aegis.
+#    \\ |#//                    <aegis.ist>
+#     .\_/.
+#
+
+# Use defaults
+export DOCKER_TLS_VERIFY=""
+export DOCKER_HOST=""
+export DOCKER_CERT_PATH=""
+export MINIKUBE_ACTIVE_DOCKERD=""
```

Comment:
Minikube might have already populated these. We want to talk to our own local docker, so we need to reset these variables first.